app/assets/javascripts/highlight.js

```diff
@@ -0,0 +1,152 @@
+function highLight() {
+  var fields = [
+      'id',
+      '_destroy',
+      'day',
+      'title',
+      'description',
+      'start_time',
+      'end_time',
+      'image',
+      'image_title'
+    ], deletedFields = [
+      'id',
+      '_destroy'
+    ];
+  function removeHighlight(ele) {
+    var deleteHighlight = confirm('Are you sure?');
+    ele = ele.closest('li')
+    if (deleteHighlight) {
+      try {
+        var highlightIndex = ele.find('input[name*=index]')[0].value;
+        createDeletedFields(ele, highlightIndex);
+      } catch (e) {
+        console.log({
+          error: e,
+          message: 'You just deleted a highlight that has not been saved'
+        });
+      } finally {
+        ele.remove();
+      }
+    }
+  }
+  function editHighlight(ele) {
+    $('#highlight_field_new').toggleClass('hide').show();
+    var values = fetchValues(ele);
+    populateFields(values);
+    ele.closest('li').remove();
+  }
+  function createDeletedFields(ele, highlight_count) {
+    var data = {};
+    data.id = ele.find('[name*=id]').val();
+    data._destroy = 1;
+    $('#event_highlights').append(deletedHighlightHtml(data, highlight_count));
+  }
+  function fetchValues(ele) {
+    var collapsibleHeader = ele.closest('.collapsible-header');
+    saveImage(collapsibleHeader);
+    return _.map(fields, function (name) {
+      return [
+        name,
+        collapsibleHeader.siblings('.ha').find('[name*=' + name + ']')[0].value
+      ];
+    });
+  }
+  function populateFields(collapsibleData) {
+    var collapsibleData = _.fromPairs(collapsibleData), hfield = $('#highlight_field_new').children(), tag = '.highlight_';
```

Comment:
`collapsibleData` is already defined as a parameter to the function, so remove the `var`  keyword at line 56